### PR TITLE
Improve the gallery example for datetime inputs

### PR DESCRIPTION
**Pull request by michaelgrund**
Fixed typo.

### CODE REVIEW

**Comment by seisman** [comment]
Could you please also change "arguments" to "parameters", following #886?

**Comment by michaelgrund** [comment]
> 
> 
> Could you please also change "arguments" to "parameters", following #886?

done!

**Comment by seisman** [comment]
The example says:

> The region parameter has to include the 𝑥 and 𝑦 axis limits as str in the form *date_min/date_max/ymin/ymax*.

Since #562, we can directly pass datetime to `region`. For example:

```
import pygmt
import datetime

fig = pygmt.Figure()

fig.basemap(
    projection="X15c/5c", 
	region=[datetime.date(2010, 1, 1), datetime.date(2030, 6, 1), 0, 10], 
	frame=["WSen", "af"]
)
fig.show()
```

Please improve the example.

**Comment by michaelgrund** [comment]
> 
> 
> The example says:
> 
> > The region parameter has to include the 𝑥 and 𝑦 axis limits as str in the form _date_min/date_max/ymin/ymax_.
> 
> Since #562, we can directly pass datetime to `region`. For example:
> 
> ```
> import pygmt
> import datetime
> 
> fig = pygmt.Figure()
> 
> fig.basemap(
>     projection="X15c/5c", 
> 	region=[datetime.date(2010, 1, 1), datetime.date(2030, 6, 1), 0, 10], 
> 	frame=["WSen", "af"]
> )
> fig.show()
> ```
> 
> Please improve the example.

Modified the description.

File: examples/gallery/plot/datetime-inputs.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
...
 
-We can pass datetime inputs based on one of the types listed above directly to the ``x`` and ``y`` arguments
-of e.g. the :meth:`pygmt.Figure.plot` method:
+We can pass datetime inputs based on one of the types listed above directly to the ``x`` and ``y`` parameters
+of e.g. the :meth:`pygmt.Figure.plot` method.
 
-The ``region`` argument has to include the :math:`x` and :math:`y` axis limits as *str* in the form 
-*date_min/date_max/ymin/ymax*.
+The ``region`` parameter has to include the :math:`x` and :math:`y` axis limits in the form 
+``[date_min, date_max, ymin, ymax]``. Here *date_min* and *date_max* can be directly defined as datetime input.
```
Comment (seisman) [inline examples/gallery/plot/datetime-inputs.py:17]:
```suggestion
[*date_min*, *date_max*, *ymin*, *ymax*]. Here *date_min* and *date_max* can be directly defined as datetime input.
```

**Review by seisman — COMMENTED** [review]
Could you also please wrap the long lines to 80 chars?

File: examples/gallery/plot/datetime-inputs.py [hunk 2]
```diff
@@ -7,14 +7,15 @@
 - :class:`numpy.datetime64`
 - :class:`pandas.DatetimeIndex`
 - :class:`xarray.DataArray`: datetimes included in a *xarray.DataArray*
-- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>`__  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``)
+- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>``__  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``)
```
Comment (seisman) [inline examples/gallery/plot/datetime-inputs.py:10]:
```suggestion
- raw datetime strings in `ISO format <https://en.wikipedia.org/wiki/ISO_8601>`__
  (e.g. ``"YYYY-MM-DD"``, ``"YYYY-MM-DDTHH"``, and ``"YYYY-MM-DDTHH:MM:SS"``) 
```

Comment (michaelgrund) [reply examples/gallery/plot/datetime-inputs.py]: Thanks for fixing @seisman. Had some trouble with formatting the URL.

**Review by seisman — APPROVED** [review]

**Pull request merged**